activities.json

```diff
@@ -933,6 +933,19 @@
     "title": "Mixed Content",
     "url": "https://w3c.github.io/webappsec-mixed-content/"
   },
+  {
+    "ciuName": null,
+    "description": "The Network Error Logging spec enables website to declare a reporting policy that can be used to report encountered network errors that prevented it from successfully loading its requested resources.",
+    "id": "network-error-logging",
+    "mdnUrl": "https://developer.mozilla.org/en-US/docs/Web/HTTP/Network_Error_Logging",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1145235",
+    "mozPosition": "negative",
+    "mozPositionDetail": "The API allows access to new kinds of network errors in a way that may leave private user info exposed. In particular, we question the value of collecting this information on a per-user basis when there is no framework for consent or transparency. Furthermore, the specification does not seem to track changes to the Reporting API it builds upon and seems effectively unmaintained.",
+    "mozPositionIssue": 99,
+    "org": "W3C",
+    "title": "Network Error Logging",
+    "url": "https://www.w3.org/TR/network-error-logging/"
```

Comment:
```suggestion
    "url": "https://w3c.github.io/network-error-logging/"
```

Reply:
wasn't sure if we want the standards track doc or the editors draft. changed, thanks.